depends/common/libpng/0001-fix-compile-if-libm-not-found.patch

```diff
@@ -0,0 +1,13 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -45,6 +45,9 @@ endif()
+ 
+ if(UNIX AND NOT APPLE AND NOT BEOS AND NOT HAIKU)
+   find_library(M_LIBRARY m)
++  if(NOT M_LIBRARY)
++    set(M_LIBRARY "")
++  endif()
```

Comment:
The `libm` was not found by him on Raspberry build. Thought to leave away as it is also added by ffmpeg, but not really like it.

Reply:
I think it is better like this. Then it's explicit.